libs/elements/src/select/vl-select.element.ts

```diff
@@ -108,26 +116,24 @@ export class VlSelect extends vlFormValidationElement(BaseElementOfType(HTMLSele
         return 'data-vl-select-dressed';
     }
 
-    _successChangedCallback(oldValue: string, newValue: string) {
+    /**
+     *
+     * @param oldValue {String} - niet in gebruik
+     * @param newValue {String}
+     */
+    _successChangedCallback(_: string, newValue: string) {
```

Comment:
ook al is het niet in gebruik, behoud mss hier ook dezelfde benaming als we in andere oude files doen, dus `oldValue` ipv `_`

Reply:
ook  voor variabelen dit niet in gebruik zijn?

=>  als ik `oldValue` schrijft maar niet gebruikt, dan `typescript` ga klagen `'oldValue' is declared but its value is never read ..`

=>  als ik `_` inzet `typescript` begrijpt mijn intentie `verklaard maar niet in gebuikt`

mag ik gewoon `oldValue` verwijderen het wordt toch en het zou duidelijker zijn wat de bedoeling is en hoe de functie moet worden gebruikt ?

Reply:
dat van `_` wist ik niet - good to know!

het is inderdaad ambetant dat gezien we de eerste variabele niet gebruiken, we die toch moeten declareren (zover ik weet) maar zou wel zeggen, als je JSDocs update / behoudt - dat die wel consistent is met de code

@ksvlaanderen kan je adviseren?

Reply:
ik heb gewoon `oldValue` daar gelaten